src/Test.UnitTests.Sarif/Core/SarifLogTests.cs

```diff
@@ -276,59 +276,68 @@ public async Task SarifLog_PostFile_PostTests()
+                .Returns(stream.ToArray());
+            httpMock.Mock(HttpMockHelper.CreateOKResponse(httpContent));
+            logPosted = await SarifLog.Post(postUri,
+                                            filePath,
+                                            fileSystem.Object,
+                                            httpClient: new HttpClient(httpMock));
+            logPosted.Item1.Should().BeTrue("with error level ToolExecutionNotifications");
+            logPosted.Item2.Should().Contain("status code 'OK'");
+            logPosted.Item2.Should().Contain(content);
+
+            stream = (MemoryStream)CreateSarifLogStreamWithToolExecutionNotifications(FailureLevel.Warning);
             fileSystem
                 .Setup(f => f.FileReadAllBytes(It.IsAny<string>()))
-                .Returns(steam.ToArray());
-            httpMock.Mock(HttpMockHelper.CreateOKResponse());
-            logPosted = await SarifLog.Post(postUri: postUri,
-                                    filePath,
-                                    fileSystem.Object,
-                                    httpClient: new HttpClient(httpMock));
-            logPosted.Should().BeFalse("with warning level ToolExecutionNotifications");
-
-            steam = (MemoryStream)CreateSarifLogStreamWithToolExecutionNotifications(FailureLevel.Error);
+                .Returns(stream.ToArray());
+            logPosted = await SarifLog.Post(postUri,
+                                            filePath,
+                                            fileSystem.Object,
+                                            httpClient: new HttpClient(httpMock));
+            logPosted.Item1.Should().BeFalse("with warning level ToolExecutionNotifications");
+            logPosted.Item2.Should().Contain("was skipped");
+
+            stream = (MemoryStream)CreateSarifLogStreamWithToolExecutionNotifications(FailureLevel.Error);
             fileSystem
                 .Setup(f => f.FileReadAllBytes(It.IsAny<string>()))
-                .Returns(steam.ToArray());
+                .Returns(stream.ToArray());
             httpMock.Mock(HttpMockHelper.CreateBadRequestResponse());
-            logPosted = await SarifLog.Post(postUri: postUri,
-                                    filePath,
-                                    fileSystem.Object,
-                                    httpClient: new HttpClient(httpMock));
-            logPosted.Should().BeTrue("with error level ToolExecutionNotifications, but the server returns BadRequest");
+            logPosted = await SarifLog.Post(postUri,
+                                            filePath,
+                                            fileSystem.Object,
+                                            httpClient: new HttpClient(httpMock));
+            logPosted.Item1.Should().BeFalse("the server returns a BadRequest even though there are error level ToolExecutionNotifications");
```

Comment:
>the server returns a BadRequest even though there are error level ToolExecutionNotifications [](http://example.com/codeflow?start=46&length=92)

the new message will be like:
`Expected logPosted.Item1 to be false because the server returns a BadRequest even though there are error level ToolExecutionNotifications, but found True.`

Reply:
Try this:

"a status code of 'BadRequest' should fail the call even though we posted error-level 'ToolExcecutionNotifications'".

For a fully constructed line like so:

`Expected logPosted.Item1 to be false because a status code of 'BadRequest' should fail the call even though we posted error-level 'ToolExcecutionNotifications', but found True.`

Reply:
Look better! Updated.